### DISCUSSION TITLE
Crash ` edyn::triangle_mesh::get_triangle_vertices (this=<optimized out>, tri_idx=<optimized out>)`

### DISCUSSION THREAD

**Discussion by frutas-fruit** [xissburg/edyn#98] [General]

**Comment by xissburg** [comment]
Are you compiling with optimizations? Can you disable optimizations and put a breakpoint there to see what are the contents of m_indices?

**Reply by xissburg** [reply]
That error means there's an invalid entity somewhere.

**Reply by xissburg** [reply]
Yeah that could be the source of the crash. It is definitely not legal to emplace the same component twice. It will trigger an assertion in EnTT. Undefined behavior does not necessarily lead to a crash.

**Reply by xissburg** [reply]
You need to obtain more extensive debugger output to better understand what is going on. It might even be something somewhere else that's causing this.
